package.json

```diff
@@ -11,13 +11,14 @@
     "node": ">=10"
   },
   "scripts": {
-    "start": "tsdx watch",
-    "build": "tsdx build",
+    "start": "npm run generate && tsdx watch",
+    "build": "npm run generate && tsdx build",
     "test": "tsdx test",
     "lint": "tsdx lint",
     "prepare": "tsdx build",
     "size": "size-limit",
-    "analyze": "size-limit --why"
+    "analyze": "size-limit --why",
+    "generate": "graphql-codegen --config codegen.yml"
```

Comment:
I would suggest same naming as in the dashboard 
https://github.com/mirumee/saleor-dashboard/blob/master/package.json#L231